ports/raspberrypi/boards/bradanlanestudio_explorer_rp2040/pins.c

```diff
@@ -38,7 +38,7 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
 
     { MP_ROM_QSTR(MP_QSTR_GP0), MP_ROM_PTR(&pin_GPIO0) },
     { MP_ROM_QSTR(MP_QSTR_GP1), MP_ROM_PTR(&pin_GPIO1) },
-    // GPIO0 and GPIO1 are also the UART
+    // GPIO0 and GPIO1 are used for IR and not for  UART
     { MP_ROM_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO0) },
     { MP_ROM_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_GPIO1) },
```

Comment:
How about renaming these, because `TX` and `RX` are traditionally used for UART? On CPX and other boards, we use these names:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_IR_TX), MP_ROM_PTR(&pin_GPIO0) },
    { MP_ROM_QSTR(MP_QSTR_IR_ RX), MP_ROM_PTR(&pin_GPIO1) },
```